src/actions.js

```diff
@@ -77,14 +80,13 @@ function getGroup(resource, action) {
                 data,
                 statusCode,
             },
+            error: false,
         }),
         [`${action}_FAILURE`]: `${resource}_${action}_FAILURE`,
         [`${funcName}Failure`]: (statusCode, data) => ({
             type: `${resource}_${action}_FAILURE`,
-            payload: {
-                data,
-                statusCode,
-            },
+            payload: new Error({ statusCode, data }),
```

Comment:
Error only takes a string argument. Creating an error here will create a stack trace starting from here and I don't think we want that.
I suggest we use a simple object here.